compiler/passes/convert-uast.cpp

```diff
@@ -392,6 +392,34 @@ struct Converter {
     return new CallExpr(PRIM_ERROR);
   }
 
+  // Used for converting special tokens in the import/use lists: for example,
+  // "align" can be used in an import statement, but is actually chpl_align.
+  Expr* reservedWordRemapForIdentInUseImport(UniqueString name) {
```

Comment:
How about calling `reservedWordRemapForIdent` here? You might have a conditional to check for the cases we want to allow in import/use lists, so might still list the cases here, but what they turn into could be consolidated into one spot.

Reply:
Well, my feeling is that `reservedWordRemapForIdent` transforms directly to `SymExpr`s where possible, which is sure to break the use/import case. If at some point we, say, change `owned` handling to also produce a (resolved) `SymExpr`, that would break `reservedWordRemapForIdentInUseImport` if it relied on `reservedWordRemapForIdent`.

Reply:
Good point. Can we factor out the UniqueString -> UniqueString mapping into one function that they can both call?

Reply:
I've made `reservedWordRemapForIdent` fall back to `reservedWordRemapForIdentInUseImport` (renaming it).  The reason is that the latter provides "unresolved" fallbacks, and if `reservedWordRemapForIdent` needs to do something, it can insert that behavior before the fallback without influencing uses and imports.